src/main/java/pl/byczazagroda/trackexpensesappbackend/dto/UpdateFinancialTransactionDTO.java

```diff
@@ -0,0 +1,15 @@
+package pl.byczazagroda.trackexpensesappbackend.dto;
+
+import org.springframework.format.annotation.DateTimeFormat;
+
+import javax.validation.constraints.Digits;
+import javax.validation.constraints.Positive;
+import javax.validation.constraints.Size;
+import java.math.BigDecimal;
+import java.time.Instant;
+
+public record UpdateFinancialTransactionDTO(
+        @Positive @Digits(integer = 15, fraction = 2) BigDecimal amount,
```

Comment:
Please rafactor this part to:
- annotation `@Positive` to `@PositiveOrZero`
- `@Digits(integer = 15, fraction = 2)` to `@Digits(integer = 13, fraction = 2)` // why exactly `13,2' instead of `15,2` - please ask on #os-backend Discord channel 
- `@DateTimeFormat(pattern = "yyyy-mm-dd hh:mm:ss")` // I will check this pattern format and update my comment as soon as possible

Reply:
Corrected